src/partials/id_verification_failed.hbs

```diff
@@ -2,18 +2,22 @@
   <div class="org-logo">
     <img class="org-logo__image" src="{{logo}}" alt="Company Logo" />
   </div>
-  <div class="icon-feedback icon-feedback--timeout icon-feedback--error">
-    <span class="icon-feedback__icon pingicon-ghost-error"></span>
-    <p class="icon-feedback__label">
-      <p class="icon-feedback--error">Verification Failed</p>
-    </p>
+  <div class="icon-feedback icon-feedback--timeout icon-feedback--error" style="margin-bottom:0">
```

Comment:
Same comment here. We use the end-user library to format these errors. If it looks different in the PF template, then we might need to revisit that

Reply:
Replaced inline style with class in main.scss;
The requirement asks to display the error icon with no error title in the failed screen.